config/database.yml

```diff
@@ -20,3 +20,9 @@ production:
 
 cucumber:
   <<: *test
+
+cucumber:
```

Comment:
is the code repetition necessary here?

Reply:
probably not. think we are in version hell.